MarshalTests/MarshalTests.swift

```diff
@@ -37,6 +39,12 @@ class MarshalTests: XCTestCase {
     
     func testBasics() {
         self.measure {
+            let one: Int = try! self.object.value(for: "dict.one")
+            XCTAssertEqual(one, 1)
+        	let valTwo: Int = try! self.object.value(for: "val\\.two")
```

Comment:
More tabs here and two lines below.

Reply:
Derp, sorry about that. BBEdit’s Detab feature failed me.